code/modules/surgery/bodyparts/bodyparts.dm

```diff
@@ -476,7 +476,7 @@
 /obj/item/bodypart/chest
 	name = BODY_ZONE_CHEST
 	desc = "It's impolite to stare at a person's chest."
-	icon_state = "default_human_chest"
+	//icon_state = "default_human_chest"
```

Comment:
I preserve this in a comment because its a *really* bad way to have it. This NEEDS to be fixed in a proper kapulimbs port. 

However bodyparts default to their main dmi file, greyscaled parts, which isnt what these icon_states are trying to point at, nor can I just put the proper icons in the greyscale folder (it would fuck it up).

Fun stuff, really.